core-services/secret-mirroring/_mapping.yaml

```diff
@@ -482,3 +482,45 @@ secrets:
   to:
     namespace: test-credentials
     name: build-github-ssh-privatekey
+- from:
+    namespace: rhcos
+    name: coreos-assembler-aliyun-secret
+  to:
+    namespace: ci
+    name: coreos-assembler-aliyun-secret
+- from:
+    namespace: rhcos
+    name: coreos-assembler-aws-cn-build-credentials
+  to:
+    namespace: ci
+    name: coreos-assembler-aws-cn-build-credentials
```

Comment:
If these were copied from the internal PSI cluster, they are likely expired.  We don't have proper access to AWS China at this time.

Reply:
Not a problem :) The secret is defined and once we update it on `api.ci` it will be automagically mirrored.